### ISSUE TITLE
Questions about running Carla on GPU servers

### ISSUE THREAD

**Issue by YilinGao-SHU** [opendilab/InterFuser#83]
Thanks so much for your work! But I got some puzzle problems when I try to run Carla on a GPU Server with 8 GPUs, when I use the following commands to start a Carla server, all the process run on gpu 0 like this. 
`CUDA_VISIBLE_DEVICES=0 ./CarlaUE4.sh --world-port=20000 -carla-server &
CUDA_VISIBLE_DEVICES=1 ./CarlaUE4.sh --world-port=20002 -carla-server &`
Have u ever met this mistake?
![image](https://github.com/opendilab/InterFuser/assets/98305137/63df075b-8ad0-44b7-b6ec-a1ca04733a19)

**Comment by deepcs233** [comment]
Hi!
Could you show the output of `nvidia-smi`? I can not understand the screenshot :)

**Comment by YilinGao-SHU** [comment]
> Hi! Could you show the output of `nvidia-smi`? I can not understand the screenshot :)

Of course! The usage of the GPUs looks like this. It can be found that all Carla process run on GPU 0.
![image](https://github.com/opendilab/InterFuser/assets/98305137/46cd9bd5-0a30-4485-baf3-77da0fcff065)
Looking forward to your reply~

**Comment by deepcs233** [comment]
Hi, this is my `nvidia-smi` output and it looks ok:
<img width="560" alt="image" src="https://github.com/opendilab/InterFuser/assets/17512647/4c0ff27c-899a-4a04-805e-e22cc59cf825">

the corresponginf script:
```bash
CUDA_VISIBLE_DEVICES=0 ./CarlaUE4.sh --world-port=2000 -opengl  &
sleep 1
CUDA_VISIBLE_DEVICES=1 ./CarlaUE4.sh --world-port=2002 -opengl  &
sleep 1
CUDA_VISIBLE_DEVICES=2 ./CarlaUE4.sh --world-port=2004 -opengl  &
sleep 1
CUDA_VISIBLE_DEVICES=3 ./CarlaUE4.sh --world-port=2006 -opengl  &
sleep 1
```

**Comment by YilinGao-SHU** [comment]
Thanks so much! I will have a try!
